.vscode/tasks.json

```diff
@@ -8,6 +8,23 @@
             "type": "shell",
             "command": "uv sync --all-extras --all-groups"
         },
+        {
+            "label": "Regenerate UI",
+            "type": "shell",
+            "linux": {
+                "command": "${command:python.interpreterPath} '${workspaceFolder}/tools/uic_wrapper.py' '${workspaceFolder}' --all-ui-files"
```

Comment:
This could just be `pre-commit run qtui -a`. Wouldn't even need to change the wrapper file too.

Reply:
that'd require pre-commit to be locally installed which might not always be the case right now i think?
but also do we want to always git add them? ive had some annoyances with that earlier

Reply:
pre-commit should always be available, if not it's an uv call away